src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Queries/Applications/GetApplications/GetApplicationsQueryHandler.cs

```diff
@@ -27,8 +27,8 @@ public async Task<GetApplicationsQueryResult> Handle(GetApplicationsQuery reques
         var totalApplicationCount = applicationsTask.Result.Applications.Count;
         var applicationList = applicationsTask.Result.Applications.Where(x =>
             x.Status == request.Status.ToString()
-            || (request.Status == ApplicationStatus.Submitted && x.Status == ApplicationStatus.Withdrawn.ToString())
-            || (request.Status == ApplicationStatus.Draft && x.Status == ApplicationStatus.Expired.ToString())
+            || (request.Status == ApplicationStatus.Submitted && x.Status == nameof(ApplicationStatus.Withdrawn))
+            || (request.Status == ApplicationStatus.Draft && x.Status == nameof(ApplicationStatus.Expired))
```

Comment:
I _feel_ like we could have problems here if someone was to rename the enum 🤔

Reply:
Makes sense, i will revert that change

Reply:
PR Comments addressed